elyra/templates/kfp/kfp_template.jinja2

```diff
@@ -66,128 +60,10 @@ def create_pipeline():
-        else:
-            auth_session["is_secured"] = True
-
-        ################
-        # Get Dex Login URL
-        ################
-        redirect_url_obj = urlsplit(auth_session["redirect_url"])
-
-        # if we are at `/auth?=xxxx` path, we need to select an auth type
-        if re.search(r"/auth$", redirect_url_obj.path):
-            # default to "staticPasswords" auth type
-            redirect_url_obj = redirect_url_obj._replace(
-                path=re.sub(r"/auth$", "/auth/local", redirect_url_obj.path)
-            )
-
-        # if we are at `/auth/xxxx/login` path, then no further action is needed (we can use it for login POST)
-        if re.search(r"/auth/.*/login$", redirect_url_obj.path):
-            auth_session["dex_login_url"] = redirect_url_obj.geturl()
-
-        # else, we need to be redirected to the actual login page
-        else:
-            # this GET should redirect us to the `/auth/xxxx/login` path
-            resp = s.get(redirect_url_obj.geturl(), allow_redirects=True)
-            if resp.status_code != 200:
-                raise RuntimeError(
-                    f"HTTP status code '{resp.status_code}' for GET against: {redirect_url_obj.geturl()}"
-                )
-
-            # set the login url
-            auth_session["dex_login_url"] = resp.url
-
-        ################
-        # Attempt Dex Login
-        ################
-        resp = s.post(
-            auth_session["dex_login_url"],
-            data={"login": username, "password": password},
-            allow_redirects=True
-        )
-        if len(resp.history) == 0:
-            raise RuntimeError(
-                f"Login credentials were probably invalid - "
-                f"No redirect after POST to: {auth_session['dex_login_url']}"
-            )
-
-        # store the session cookies in a "key1=value1; key2=value2" string
-        auth_session["session_cookie"] = "; ".join([f"{c.name}={c.value}" for c in s.cookies])
-
-    return auth_session
-{% endif %}
-
-
 if __name__ == "__main__":
 
-{% if kf_secured %}
-    # user id and password are required parameters
-    if(len(sys.argv) < 3):
-        print(f'Invocation: python {sys.argv[0]} '
-               '<kf-user-id> <kf-password>')
-        sys.exit(1)
-
-    print('Trying to authenticate with Kubeflow server ...')
-
-    api_endpoint = '{{ api_endpoint }}'.rstrip('/')
-    api_username = sys.argv[1]
-    api_password = sys.argv[2]
-
-    try:
-        # attempt to create a session cookies using the provided `api_username` and `api_password`
-        auth_session = get_istio_auth_session(
-            url=api_endpoint,
-            username=api_username,
-            password=api_password
-        )
-    except Exception as ex:
-        raise RuntimeError(
-            f"Failed to create istio auth session for Kubeflow endpoint: '{api_endpoint}'"
-        ) from ex
-
-    auth_cookie = auth_session['session_cookie']
-{% else %}
+    # TODO authentication
```

Comment:
What is this line?

Reply:
Pipeline export to Python DSL is currently not supported (and thus disabled). We have not seen any evidence that anybody actually used that feature in a long time. I didn't want to invest time adding jinja templates for the authentication code if it is decided to permanently remove this feature.  The TODO is just a reminder that the template does not generate any authentication code.

Users that want to export pipelines can still use the YAML format as an alternative.